utils/watchcat/files/watchcat.sh

```diff
@@ -137,12 +137,12 @@ watchcat_monitor_network() {
 		for host in $ping_hosts; do
 			if [ "$iface" != "" ]; then
```

Comment:
ummm, why not just use -n ?

Reply:
12 days ago you have broken watchcat script. Kofec tried to repair it. Do it better if you want.

Reply:
I did?

Reply:
"[jtkohl](https://github.com/openwrt/packages/commits?author=jtkohl) authored and [neheb](https://github.com/openwrt/packages/commits?author=neheb) committed 12 days ago"

yes, you are right. it was tooth fairy. for sure.

Reply:
@neheb You can change it if you think that it is better solution.

Reply:
> ummm, why not just use -n ?

That would be cleaner code.  Note, howevver, that line isn't related to the defect introduced with the added IPv6 support.  It's been the same since before those changes, so I'd say don't touch that right now, submit a separate PR if you'd like to clean that up.